includes/class-wc-stripe-helper.php

```diff
@@ -1491,16 +1491,6 @@ public static function payment_method_allows_manual_capture( string $payment_met
 		return in_array( $payment_method_id, [ 'stripe', 'stripe_affirm', 'stripe_klarna', 'stripe_afterpay_clearpay' ], true );
 	}
 
-	/**
-	 * Verifies if the provided order contains the identifier for a wallet method.
-	 *
-	 * @param WC_Order $order The order.
-	 * @return bool
-	 */
-	public static function is_wallet_payment_method( $order ) {
```

Comment:
@wjrosa because this is a public function, we'll need to deprecate it.

Reply:
Oops, I forgot about this again 🤦 . Done in https://github.com/woocommerce/woocommerce-gateway-stripe/pull/3585/commits/6517af25e6704e96d993727656ac06bc6dab6896